.buildkite/pipeline.yml

```diff
@@ -1,5 +1,10 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+env:
+  ASDF_MAGE_VERSION: '1.15.0'
 
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: "Generate dynamic pipeline"
+    command: ".buildkite/scripts/generate_pipeline.sh"
+    agents:
+      provider: "gcp"
+      image: "family/platform-ingest-beats-ubuntu-2204"
```

Comment:
Let's use a smaller machine than the default (to reduce costs) until we switch to a custom container image. e.g.:

`machineType: "n2-standard-2"`

Reply:
I think we can use a container here, right?